src/lib/Form/Type/ReferenceCollectionType.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class ReferenceCollectionType extends AbstractType
+{
+    /** @return class-string<\Symfony\Component\Form\AbstractType> */
```

Comment:
Nitpick: This shouldn't really be necessary, since Symfony does provide return type for this:
https://github.com/symfony/symfony/blob/706d8628e52f7d912077ecd03ea3ca4ecbf841ac/src/Symfony/Component/Form/FormTypeInterface.php#L84

```suggestion
```